tests/sentry/backup/test_imports.py

```diff
@@ -739,6 +741,19 @@ def test_global_import_scoping(self):
             == RegionImportChunk.objects.values("import_uuid").first()
         )
 
+    def test_global_import_initial_model_deletion(self):
```

Comment:
Suggestion on how we can rewrite this test to be a bit stronger:

- Instead of importing a fresh-install, import an empty JSON: `[]`.
- Check that all tables mentioned as not being `region: Excluded` in the output of `sentry.backup.dependencies.dependencies()` are, in fact, empty and have had their seq numbers reset
- Export, and check that the export is empty (`[]`) as well.

This will give us good coverage of the invariant we want to test: that the tables of all models that an import may touch are fully reset. Because the models you have on L751-755 are especially important (they are what motivated this whole feature!), I'd still keep those specific checks, with a comment above them explaining why you are double-checking them in particular. This will ensure that even if the output of `dependencies()` gets wonky or corrupted in the future, we'll still have a canary for the most important tables.

If you still want to test the specific case that motivated the bug, that there are default projects in particular, we could add another test where we create default projects as we do here.

Reply:
done